core/src/test/java/se/tre/freki/storage/StoreTest.java

```diff
@@ -76,27 +75,4 @@ public void testGetNameMissingAbsent() throws Exception {
     final Optional<String> missing = store.getName(missingLabelId(), LabelType.TAGV).get();
     assertFalse(missing.isPresent());
   }
-
-  @Test
-  @Ignore
-  public void testRenameIdFoundOnNewName() throws Exception {
-    final LabelId id = store.createLabel("name", LabelType.TAGK).get();
-    store.renameLabel("newname", id, LabelType.TAGK).get();
-    final LabelId newNameId = store.getId("newname", LabelType.TAGK).get().get();
-    assertEquals(id, newNameId);
-  }
-
-  @Test
-  @Ignore
-  public void testRenameIdNotFoundOnOldName() throws Exception {
```

Comment:
This test should not be removed either. It tests that the `Store` does not find the old name after a successful call to `#renameLabel`.